src/index.css

```diff
@@ -28,18 +29,16 @@ h2 {
 
 h3 {
   font-size: 1.5rem;
+  padding-bottom: 0.5rem !important;
 }
 
 table, th, td, tr {
   font-family: "Consolas";
   font-size: 1.125rem;
 }
 
-.white-text {
-  color: var(--clr-white)
-}
-
 body, input, textarea, button {
+  color: var(--clr-white);
```

Comment:
- All H1 and H3 will have their own specific padding unless specified otherwise
- All text will be white unless specified otherwise